vendor/vendor.json

```diff
@@ -173,4 +173,4 @@
 			"revisionTime": "2015-06-24T11:29:02+01:00"
 		}
 	]
-}
+}
```

Comment:
Yeah, `govendor` unfortunately doesn't place newlines at the end of this file :( The only reason why it had a newline before is that I manually edited it for some other reasons. Maybe we should fix that upstream?